layouts/compmech.h

```diff
@@ -0,0 +1,29 @@
+
+constexpr int R = 5, C = 19;
+
+// The row pins, to be modified according to the exact wiring.
+const int row_pins[R] = { 20, 21, 22, 23, 24 };
+
+// The column pins, to be modified according to the exact wiring.
+const int col_pins[C] = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19};
+
+Layer<R,C> layers[] = {
+  { "qwerty", l0, l1, "",
```

Comment:
Since you don't have a 'fn' key, this line can be simplified to "qwerty", l0, l0, "".